charts/datadog/templates/_container-host-release-volumemounts.yaml

```diff
@@ -2,12 +2,10 @@
 {{- if eq (include "should-enable-system-probe" .) "true" }}
 - name: os-release-file
   mountPath: /host{{ .Values.datadog.systemProbe.osReleasePath | default .Values.datadog.osReleasePath }}
-  mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
```

Comment:
the option needs to be removed from the `values.yaml` file: https://github.com/DataDog/helm-charts/blob/2f7e78ad860157f64d6e6d7ccb86ffe27a38c440/charts/datadog/values.yaml#L88

Reply:
It is still used in a lot of places.